api/src/main/java/io/opentelemetry/api/trace/TraceState.java

```diff
@@ -69,6 +70,9 @@ static TraceStateBuilder builder() {
   /** Iterates over all the key-value entries contained in this {@link TraceState}. */
   void forEach(BiConsumer<String, String> consumer);
 
+  /** Returns a read-only view of this {@link TraceState} as a {@link Map}. */
+  Map<String, String> asMap();
```

Comment:
So... hard question. Do we need this now, or could we add this to the API later, when needed? We should definitely limit the public API as much as possible, and I'd like to understand the use-case for this.